kubeflow/trainer/rhai/traininghub.py

```
@@ -38,33 +55,409 @@ class TrainingHubTrainer:
+                    if throughput > 0 and remaining_steps > 0
+                    else None
+                )
+
+            loss_val = metrics.get("avg_loss", 0)
+            lr_val = metrics.get("lr")
+            grad_norm_val = metrics.get("gradnorm")
+            throughput_val = metrics.get("overall_throughput")
+
+            return {
+                {
+                    "progressPercentage": int(round(percent)),
```

Ruff (0.14.7)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)